### PR TITLE
test: fix failing cypress tests 

### PR DESCRIPTION
**Pull request by Stofftiger**
fixed cypress tests for:
- product-configurator-vc-cart.e2e-flaky 
- product-configurator-vc-ov-filter-menu-e2e-spec-flaky
- product-configurator-textfield.e2e

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts [hunk 1]
```diff
@@ -113,8 +120,19 @@ function completeDigitalCameraConfiguration() {
   configurationVc.selectAttributeAndWait('CAMERA_PIXELS', RB, 'P16');
   configurationVc.selectAttributeAndWait('CAMERA_SENSOR', RB, 'F');
   configurationVc.selectAttributeAndWait('CAMERA_VIEWFINDER', RB, 'R');
```
Comment (Uli-Tiger) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts]: we should also wait for pricing here

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts [hunk 2]
```diff
@@ -113,8 +120,19 @@ function completeDigitalCameraConfiguration() {
...
+
+  configurationVc.selectAttributeAndWait(
+    'CAMERA_SD_CARD',
+    CBL,
+    'SDXC',
+    isPricingEnabled
+  );
   configurationVc.selectAttributeAndWait('CAMERA_SECOND_SLOT', RB, 'Y');
   configurationVc.selectAttributeAndWait('CAMERA_FORMAT_PICTURES', RB, 'RAW');
   configurationVc.selectAttributeAndWait('CAMERA_MAX_ISO', RB, '25600');
```
Comment (Uli-Tiger) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts]: same here, wait for pricing

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts [hunk 3]
```diff
@@ -137,6 +155,16 @@ function completeDigitalCameraConfiguration() {
   );
 
   configurationVc.clickOnNextBtnAndWait('Options');
```
Comment (Uli-Tiger) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts:214]: same here

**Review by Uli-Tiger — CHANGES_REQUESTED** [review]
please wait for pricing on each value change and on each group navigation

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts (hunk 1 above)
Comment (Stofftiger) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts]: done

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts (hunk 2 above)
Comment (Stofftiger) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts]: done

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts (hunk 3 above)
Comment (Stofftiger) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-ov-filter-menu-e2e-spec-flaky.cy.ts:214]: done

**Review by Uli-Tiger — APPROVED** [review]

**Pull request merged**